index.html

```diff
@@ -38,6 +38,7 @@
 
   <!-- LineIcons -->
 
+  <link href='https://unpkg.com/boxicons@2.1.2/css/boxicons.min.css' rel='stylesheet'>
```

Comment:
Actually I want this scroll top button in dark/index.html and light/index.html files @Devemmy01 
No need in the index.html file